src/Domain/Security/ValueObject/AuthUser.php

```diff
@@ -50,4 +52,14 @@ public function roles(): array
         return $this->roles;
     }
 
+
+    public function changePassword(EncodedPasswordInterface $oldPassword, EncodedPasswordInterface $newPassword): void
```

Comment:
Why do you need the old password, when you want to change to password?

Reply:
For security reasons. Imagine you leave your session open in a browser used not only by you, this will avoid malicious actions.

Reply:
ok. Thanks for the explanation